frontend/src/components/MyProfile/index.tsx

```diff
@@ -45,12 +46,13 @@ const MyProfile = () => {
     <>
       <div className="user-info-section">
         <div className="user-image">
-          <AvatarImage userName={`${userData.firstName} ${userData.lastName}`} size="large" />
+          <AvatarImage userName={`${userDetails.user.firstName} ${userDetails.user.lastName}`} size="large" />
```

Comment:
For some reason, it displayed `undefined undefined` at the beginning, and later on got updated to my user name.